jsonnet/dashboards/k8s/container-overview.libsonnet

```diff
@@ -79,10 +115,11 @@ local getNextIndex(arrays) =
       local valueMaps = std.flattenArrays([valueMapsOk, valueMapsWaitingErrors, valueMapsTerminatedErrors]);
 
       local okQueries = [
-        'sum by (container, namespace, pod) (kube_pod_container_status_terminated_reason{cluster=~"$cluster", reason="Completed"} * 1)',
+        'sum by (container, namespace, pod) (kube_pod_container_status_terminated_reason{cluster=~"$cluster", pod=~"$pod", reason="Completed"} * 1)',
```

Comment:
I don't understand why, there is only `pod=~"$pod"` and not also `container=~"$container"` and `namespace=~"$namespace"`, like in [container detail](https://github.com/dNationCloud/kubernetes-monitoring/blob/main/jsonnet/dashboards/k8s/container-detail.libsonnet#L40). And maybe it should be added to all parts of `statusExpr`.

Reply:
When you add `container=~"$container"` to the query expressions and select all from drop-down menu you will get this type of error: `Unknown error during query transaction. Please check JS console logs.` This error do not show up when you select specific namespace for example.

Reply:
it seems like a grafana bug